lumibot/strategies/strategy.py

```diff
@@ -1858,7 +1858,7 @@ def get_last_price(self, asset, quote=None, exchange=None, should_use_last_close
         """
 
         # Check if the asset is valid
-        if asset is None or (type(asset) == Asset and asset.is_valid() is False):
+        if asset is None or (type(asset) is Asset and asset.is_valid() is False):
```

Comment:
isinstance() should always be used instep of ```type() is <>" checks.  Can you change this as well?

Reply:
Done